pkg/kubernetes/secrets/plugin.go

```diff
@@ -1,35 +1,48 @@
 package secrets
 
 import (
-	"os"
+	"fmt"
 
 	"get.porter.sh/plugin/kubernetes/pkg/kubernetes/config"
+	portercontext "get.porter.sh/porter/pkg/context"
 	"get.porter.sh/porter/pkg/secrets"
+	"get.porter.sh/porter/pkg/secrets/plugins"
 	cnabsecrets "github.com/cnabio/cnab-go/secrets"
 	"github.com/hashicorp/go-hclog"
-	"github.com/hashicorp/go-plugin"
+	hplugin "github.com/hashicorp/go-plugin"
+	"github.com/mitchellh/mapstructure"
+	"github.com/pkg/errors"
 )
 
-const PluginInterface = secrets.PluginInterface + ".kubernetes.secret"
+const PluginKey = plugins.PluginInterface + ".kubernetes.secrets"
 
 var _ cnabsecrets.Store = &Plugin{}
 
-// Plugin is the plugin wrapper for accessing secrets from Kubernetes Secrets.
+type PluginConfig struct {
+	KubeConfig string `mapstructure:"kubeconfig"`
```

Comment:
When/how is this used? Is it for running outside of the operator?

Reply:
This needs to go away.  It didn't get cleaned up with some other work.